sycl/test/regression/fsycl-host-compiler-win.cpp

```diff
@@ -1,5 +1,5 @@
 // RUN: %clang_cl -fsycl -fsycl-host-compiler=cl -DDEFINE_CHECK -fsycl-host-compiler-options="-DDEFINE_CHECK /std:c++17" /Fe%t1.exe %s
-// RUN: %RUN_ON_HOST %t1.exe
+// RUN: %t1.exe
 // REQUIRES: system-windows
```

Comment:
I don't understand how the test fails in CI on Linux: https://github.com/intel/llvm/runs/8267871292?check_suite_focus=true. Can you please check?